.editorconfig

```diff
@@ -116,6 +116,9 @@ dotnet_diagnostic.IDE0082.severity = warning
 # IDE0110: Remove unnecessary discard
 dotnet_diagnostic.IDE0110.severity = warning
 
+# CS8524: Switch doesn't handle unnamed enum values
+dotnet_diagnostic.CS8524.severity = none
```

Comment:
I don't like disabling this globally. If there is a reason why the warning should be suppressed please use #pragma